rigs/heft-node-rig/profiles/default/config/typescript.json

```diff
@@ -65,7 +65,7 @@
     /**
      * File extensions that should be copied from the src folder to the destination folder(s).
      */
-    "fileExtensions": [".json"]
+    "fileExtensions": [".json", ".yaml"]
```

Comment:
Do we want to add this to the rig? How often do people use YAML files in their src folder?